backend/tests/integration/search/test_facets.py

```diff
@@ -1,27 +1,34 @@
-from inspirehep.search.facets import (
-    hep_author_publications,
-    hep_author_publications_cataloger,
-    hep_conference_contributions,
-    records_hep,
-    records_hep_cataloger,
+from inspirehep.search.aggregations import (
+    conf_subject_aggregation,
+    hep_arxiv_categories_aggregation,
+    hep_author_aggregation,
+    hep_author_count_aggregation,
+    hep_collaboration_aggregation,
+    hep_collection_aggregation,
+    hep_doc_type_aggregation,
+    hep_earliest_date_aggregation,
+    hep_self_author_affiliations_aggregation,
+    hep_self_author_names_aggregation,
+    hep_subject_aggregation,
+    jobs_field_of_interest_aggregation,
+    jobs_rank_aggregation,
+    jobs_region_aggregation,
+    jobs_status_aggregation,
 )
+from inspirehep.search.facets import hep_author_publications
 
 
-def test_hep_author_publications_facets(base_app):
+def test_hep_author_publications_facets_without_exclude(base_app):
     expect = {
         "meta": {
             "order": 3,
             "title": "Collaborators",
             "type": "checkbox",
             "split": True,
         },
-        "terms": {
-            "exclude": "Jones, Jessica",
-            "field": "facet_author_name",
-            "size": 20,
-        },
+        "terms": {"field": "facet_author_name", "size": 20},
     }
-    with base_app.test_request_context("?author_recid=Jones, Jessica"):
+    with base_app.test_request_context():
```

Comment:
I'd refactor this to use api_client

Reply:
I'm not sure I understand what you mean. The reason I added the test request context is because we extract the author from the context for the hep author publications author exclusion and the self affiliations